package.json

```diff
@@ -29,10 +29,10 @@
     "gulp-bump": "^3.2.0",
     "gulp-clean": "0.4.0",
     "gulp-concat": "2.6.1",
-    "gulp-connect": "5.7.0",
+    "gulp-connect": "^5.0.0",
     "gulp-jshint": "2.1.0",
     "gulp-replace": "1.0.0",
-    "gulp-sourcemaps": "3.0.0",
```

Comment:
This is a downgrade?

Reply:
It was changed by `npm audit fix` to fix some vulnerabilities automatically.